server/index.js

```diff
@@ -48,8 +56,37 @@ app.get('/hello', (req, res) => {
     `);
 })
 
-//// Below are all of our own custom api endpoints that we can call from the frontend
+//QUESTION: do we need 'req' parameter in function, given that it is never used?
+app.get('/now-playing-movie', async (req, res) => {
+  const pageNumberStart = 1;
+  const nowPlaying = await nowPlayingMovieList(pageNumberStart);
+  const list = listPage(nowPlaying.results, 'nowPlayingMovieList');
+  res.send(list);
+});
```

Comment:
Something to be aware of is that express doesn't handle async errors.  There are a few ways to handle it.  The simplest of which is to wrap all handler content with try/catch.  

```suggestion
app.get('/now-playing-movie', async (req, res) => {
  try {
    const pageNumberStart = 1;
    const nowPlaying = await nowPlayingMovieList(pageNumberStart);
    const list = listPage(nowPlaying.results, 'nowPlayingMovieList');
    res.send(list);
  } catch (error) {
    // do some appropriate thing here
    // for example return some error page:
    res.send(errorPage);
  }
});
```

Here's an [article ](https://zellwk.com/blog/async-await-express/) that goes into more detail.